examples/helm/tiller-template.yaml

```diff
@@ -28,16 +29,19 @@ objects:
     verbs:
     - get
 
-- kind: RoleBinding
-  apiVersion: v1
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
   metadata:
-    name: tiller
-  roleRef:
-    name: tiller
+    name: edit-79929
```

Comment:
I don't like the name, do you think I can change it?

Reply:
you can name it anything you want, but per my comment below and your response, perhaps this rolebinding can go away entirely.